R/biome.R

```diff
@@ -0,0 +1,181 @@
+#' Create new biome
+#'
+#' @param core Hector core
+#' @param biome Name of new biome
+#' @param veg_c0 Initial vegetation C pool (default = `0`)
+#' @param detritus_c0 Initial detritus C pool (default = `0`)
+#' @param soil_c0 Initial soil C pool (default = `0`)
+#' @param npp_flux0 Initial net primary productivity (default = `0`)
+#' @param warmingfactor Temperature multiplier (default =
+#'   `1.0`)
+#' @param beta CO2 fertilization effect (default = `0.36`) 
+#' @return Hector core, invisibly
+#' @author Alexey Shiklomanov
+#' @export
+create_biome <- function(core, biome,
+                         veg_c0 = 0,
+                         detritus_c0 = 0,
+                         soil_c0 = 0,
+                         npp_flux0 = 0,
+                         warmingfactor = 1,
+                         beta = 0.36) {
```

Comment:
Note that Q10 is _not_ currently a biome-specific paramater, _even though we say it is_. Unlike these changes, which are all new features and do not affect existing behavior, making Q10 a biome-specific parameter would be a breaking change, so I deliberately left it out of this PR. If we agree on this design, then I'll make that change in a PR hot on the heels of this one.

Reply:
Why have default values for these parameters? Shouldn't the user be required to specify them?

Reply:
> Why have default values for these parameters? Shouldn't the user be required to specify them?

I usually air on the side of minimizing the number of things users _have_ to set to minimize user-side friction. The defaults for `warmingfactor` and `beta` are sensible values for which the intuition is not obvious, so I'm inclined to keep those. However, you're probably right that it makes sense to remove the defaults for the initial C pools and NPP, since the current defaults don't actually make sense either.

Reply:
Removed defaults for `create_biome` in 2e77808f8f1f5f7da2b1c83d6be4528d2cfb60f5.